src/main/kotlin/no/nav/familie/ef/sak/vedtak/uttrekk/OpprettUttrekkArbeidssøkerTask.kt

```diff
@@ -15,11 +19,36 @@ import java.time.YearMonth
 )
 class OpprettUttrekkArbeidssøkerTask(
         private val uttrekkArbeidssøkerService: UttrekkArbeidssøkerService,
+        private val fagsakService: FagsakService,
         private val taskRepository: TaskRepository
 ) : AsyncTaskStep {
 
+    private val logger = LoggerFactory.getLogger(javaClass)
+
     override fun doTask(task: Task) {
-        uttrekkArbeidssøkerService.opprettUttrekkArbeidssøkere()
+        val årMåned = objectMapper.readValue<YearMonth>(task.payload)
+        val uttrekk = uttrekkArbeidssøkerService.hentArbeidssøkereForUttrekk(årMåned)
+        val aktiveIdenter = fagsakService.hentAktiveIdenter(uttrekk.map { it.fagsakId }.toSet())
+
+        var feilede = 0
+        uttrekk.forEach {
+            if (uttrekkArbeidssøkerService.uttrekkFinnes(årMåned, it.fagsakId)) {
+                return@forEach
+            }
+            try {
+                uttrekkArbeidssøkerService.opprettUttrekkArbeidssøkere(årMåned = årMåned,
+                                                                       fagsakId = it.fagsakId,
+                                                                       behandlingIdForVedtak = it.behandlingIdForVedtak,
+                                                                       personIdent = aktiveIdenter[it.fagsakId]
+                                                                                     ?: error("Kunne ikke finne fagsakID"))
+            } catch (ex: Exception) {
+                logger.error(ex.message, ex)
+                ++feilede
+            }
+        }
+        if (feilede > 0) {
+            error("Kunne ikke opprette ${feilede} av ${uttrekk.size} uttrekk")
```

Comment:
Du trenger ikke curly brackets rundt feilede, det burde intellij gi warning om? 
```suggestion
            error("Kunne ikke opprette $feilede av ${uttrekk.size} uttrekk")
```